src/pages/mpl-hybrid/create-escrow.md

```diff
@@ -0,0 +1,217 @@
+- Core NFT Assets Minted to the Collection - [Link](/core/guides/javascript/how-to-create-a-core-nft-asset-with-javascript)
+- An SPL Token created with required token amount. - [Link](/guides/javascript/how-to-create-a-solana-token)
+- An online storage of sequential metadata JSON files at a consistent gateway/uri.
+
+Initializing the escrow is the essential step that links an NFT collection with a fungible token. Before starting this step, you should have ready a Core collection address, a fungible token mint address, and a range of off-chain metadata URIs using numerically named, sequential files. The need for Base URI string consistency will limit some off-chain metadata options. Note that the authority of the escrow needs to match the authority of the collection to perform metadata updates. Additionally, because the escrow is funded, there is no need to be the token authority which allows collections to be backed by existing memecoins or other fungible assets.
+
+## MPL-Hybrid Escrow Account Structure
+
+The MPL Hybrid Escrow is the heart of the program which stores all information regarding the project.
+
+{% totem %}
+{% totem-accordion title="On Chain MPL-404 Escrow Data Structure" %}
+
+The onchain account structure of an MPL-404 Escrow [Link](https://github.com/metaplex-foundation/mpl-hybrid/blob/main/programs/mpl-hybrid/src/state/escrow.rs)
+
+| Name           | Type   | Size | Description                                      |     |
+| -------------- | ------ | ---- | ------------------------------------------------ | --- |
+| collection     | Pubkey | 32   | The collection account                           |     |
+| authority      | Pubkey | 32   | The authority of the Escrow                      |     |
+| token          | Pubkey | 32   | The fungible token to be dispensed               |     |
+| fee_location   | Pubkey | 32   | The account to send token fees to                |     |
+| name           | String | 4    | The NFT name                                     |     |
+| uri            | String | 8    | The base uri for the NFT metadata                |     |
+| max            | u64    | 8    | The max index of NFTs that append to the uri     |     |
+| min            | u64    | 8    | The minimum index of NFTs that append to the uri |     |
+| amount         | u64    | 8    | The token cost to swap                           |     |
+| fee_amount     | u64    | 8    | The token fee for capturing the NFT              |     |
+| sol_fee_amount | u64    | 8    | The sol fee for capturing the NFT                |     |
+| count          | u64    | 8    | The total number of swaps                        |     |
+| path           | u16    | 1    | The onchain/off-chain metadata update path       |     |
+| bump           | u8     | 1    | The escrow bump                                  |     |
+
+{% /totem-accordion %}
+{% /totem %}
+
+## Create Escrow
+
+### Args
+
+#### name
+
+The name of your escrow.
```

Comment:
what is it important for? Maybe provide more context

Reply:
I think it has zero importance/impact to be honest other than its just a title that you can perhaps pull and display on a UI if you want to. It's not part of the seed for the escrow, not part of the naming process of released nfts (that gets changed to just "Captured".

Will add the UI name part in.